packages/backend/src/managers/applicationManager.spec.ts

```diff
@@ -1,15 +1,13 @@
 import { type MockInstance, describe, expect, test, vi, beforeEach } from 'vitest';
 import { ApplicationManager } from './applicationManager';
 import type { RecipeStatusRegistry } from '../registries/RecipeStatusRegistry';
-import type { ExtensionContext } from '@podman-desktop/api';
 import type { GitManager } from './gitManager';
 import os from 'os';
-import fs, { Stats, type Dirent } from 'fs';
-import path from 'path';
+import fs from 'fs';
```

Comment:
```suggestion
import fs from 'node:fs';
```

Reply:
using `node:` prefix is recommended for all the 'internal' packages

it's to avoid for example that you use `https://www.npmjs.com/package/fs`